dynamiq/workflow/workflow.py

```diff
@@ -16,7 +16,9 @@
 
 
 class Workflow(BaseModel, Runnable):
-    """Workflow class for managing and running workflows.
+    """
+    A container for a flow that manages its lifecycle, YAML serialization, and versioning
+    and adds workflow-specific metadata, callbacks, and configuration handling.
```

Comment:
Please little bit clean up to avoid few `and` in `and versioning and adds ...`. 
Smh like:
`A container for a flow that manages its lifecycle, YAML serialization, versioning, metadata, callbacks, and configuration`